types/jasmine/jasmine-tests.ts

```diff
@@ -777,6 +783,30 @@ describe("jasmine.arrayContaining", () => {
     });
 });
 
+describe("jasmine.arrayWithExactContents", () => {
+    var foo: Array<number>;
+
+    beforeEach(() => {
+        foo = [1, 2, 3, 4];
+    });
+
+    it("matches arrays with exactly the same values", () => {
+        expect(foo).toEqual(jasmine.arrayWithExactContents([1, 2, 3, 4]));
+        expect(foo).not.toEqual(jasmine.arrayWithExactContents([6]));
```

Comment:
Add lines for `.toBe` too.

Reply:
added `.toBe` to `jasmine.arrayWithExactContents` and `jasmine. arrayContaining `